app/templates/components/hero.html

```diff
@@ -1,14 +1,19 @@
-<div class="hero-section">
-  <h1 class="pt-4 pb-2 font-bold text-3xl">{{ translations.get("title") }}</h1>
-  <h2 class="font-bold leading-tight text-2xl mt-0 mb-2 text-pink-500">
-    {{ translations.get("disclaimer") }}
-  </h2>
-  <p class="py-4">
-    Lorem ipsum dolor sit, amet consectetur adipisicing elit. Assumenda soluta
-    sed consequatur rem saepe dolores voluptates recusandae explicabo officiis
-    ipsa nesciunt, aliquid placeat dolor consectetur ullam tenetur impedit eaque
-    quam quas esse facilis voluptate ducimus voluptatem. Itaque dolores
-    similique, eaque animi magni, quaerat consectetur sit suscipit quas nam
-    molestias culpa.
-  </p>
+<div
+    class="hero-section fixed top-0 left-0 bg-corporate-blue text-white w-screen h-3/5 -z-10 py-10 px-mobile tablet:px-desktop desktop:max-w-hero desktop:h-full"
```

Comment:
On desktop it’s not possible to highlight the text in the hero! It’s not the end of the world, but it hints at a layout issue somewhere. I couldn’t really pinpoint it to any specific style yet. 🤔

Reply:
jep. fixed it, was a negatve z-index from the overlay in mobile